torch/_numpy/random.py

```diff
@@ -42,10 +42,28 @@ def inner(*args, **kwds):
         if USE_NUMPY_RANDOM is False:
             return func(*args, **kwds)
         elif USE_NUMPY_RANDOM is True:
```

Comment:
Now that we're at it, could you make this a flag in `_dynamo.config` near the other numpy flags that are there? I forgot to do so in the upstreaming PR

Reply:
Feel free to do it in this PR or in a follow up